Tasks/VsTest/vstest.ts

```diff
@@ -485,6 +490,7 @@ function cleanFiles(responseFile: string, listFile: string, testCaseFilterFile:
 
 function deleteVstestDiagFile(): void {
     if (vstestConfig && vstestConfig.vstestDiagFile && pathExistsAsFile(vstestConfig.vstestDiagFile)) {
+        console.log('##vso[task.uploadfile]'+ vstestConfig.vstestDiagFile);   
         tl.debug('Deleting vstest diag file ' + vstestConfig.vstestDiagFile);
         tl.rmRF(vstestConfig.vstestDiagFile);
```

Comment:
Is it possible the file gets deleted before it gets uploaded. Since upload is async

Reply:
Actually I  think _temp directory gets auto cleaned after the job completes. This rmRf most likely is redundant and can be removed. Please check.

Reply:
Yes this is a possibility, we should remove the deletion of diag file